examples/advanced_usage/handle_cursor_pagination.py

```diff
@@ -0,0 +1,33 @@
+from atproto import Client
+from atproto.xrpc_client import models
+
+
+def main():
+    # This is an example for get_follows method.
+    client = Client()
+    client.login('my-handle', 'my-password')
+    handle = 'target-handle'
+
+    def _fetch(actor: str, cursor: str | None) -> models.AppBskyGraphGetFollows.Response:
```

Comment:
please note that the `|` type hint available since python 3.10. The library supports Python starting from 3.7. Please use typing.Union instead

Reply:
[bcbbd9c](https://github.com/MarshalX/atproto/pull/93/commits/bcbbd9cde0aa14b7286084d1b75b8c3adf3107de)